src/resources/postcss/admin/category-colors/admin-category.pcss

```diff
@@ -0,0 +1,194 @@
+.table-view-list.tags {
+	.fixed .column-category_priority {
+		width: 75px;
+	}
+
+	.column-category_color {
+		width: 25%;
+
+		.tec-events-taxonomy-table__category-color-preview {
+			aspect-ratio: 1/1;
+			border-radius: 6px;
+			display: inline-block;
+			width: 20px;
+		}
+	}
+}
+
+/** Quick Edit section **/
+.inline-edit-row {
+	.tec_category_colors__wrap {
+				line-height: 1;
+		label {
+			line-height: 1;
+			margin: 0;
+			padding: 2px 0;
+		}
+	}
+}
+
+.tec_category_colors__wrap {
+	line-height: 1;
+
+	.tec-events-category-colors__container {
+		container-type: inline-size;
+	}
+
+	/* 1-column layout */
+
+	.tec-events-category-colors__grid {
+		display: grid;
+		grid-template-columns: 1fr;
+		grid-template-rows: repeat(4, auto);
+		gap: 1rem;
+		grid-template-areas:
+			"color-picker-1"
+			"color-picker-2"
+			"color-picker-3"
+			"preview";
+
+		.form-field {
+			margin: 0;
+		}
+	}
+
+	.tec-events-category-colors__group:nth-child(1) {
+		grid-area: color-picker-1;
+	}
+
+	.tec-events-category-colors__group:nth-child(2) {
+		grid-area: color-picker-2;
+	}
+
+	.tec-events-category-colors__group:nth-child(3) {
+		grid-area: color-picker-3;
+	}
+
+	.tec-events-category-colors__preview {
+		grid-area: preview;
+		width: 100%;
+	}
+
+	/* Category preview styles */
+
+	.tec-events-category-colors__preview-box {
+		display: inline-block;
+		max-width: fit-content;
+		width: 100%;
+
+		span {
+			box-sizing: border-box;
+			display: inline-block;
+			max-width: 100%;
+			padding: 3px 6px;
+		}
+	}
+
+	/* WP Picker Container */
```

Comment:
I'd move these to a separate file for organization - not needed but it's tidier?